java/pom.xml

```diff
@@ -776,6 +787,11 @@
             </reports>
           </reportSet>
         </reportSets>
+        <configuration>
+          <sourceFileExcludes>
+            <sourceFileExclude>**/module-info.java</sourceFileExclude>
+          </sourceFileExcludes>
+        </configuration>
```

Comment:
We may want to document module-info.java in Java 9+ right?

Reply:
Yes, I think we should wait until JDK8 support is dropped to do this though. We run into similar issues as we ran into while compiling (need to set release to 9, but that breaks how we utilize Unsafe).